CHANGELOG.md

```diff
@@ -3,6 +3,116 @@
+- 🌎 Add Simplified Chinese (zhCN) localization (#2431) @wlf100220
+- 🌎 Add Korean (koKR) localization (#2446) @zzossig
+
+### `@mui/x-data-grid@v5.0.0-beta.0` / `@mui/x-data-grid-pro@v5.0.0-beta.0`
+
+#### Breaking changes
+
+- [DataGridPro] Remove `apiRef.current.getState` method
+
+  ```diff
+  -const state = apiRef.current.getState();
+  +const state = apiRef.current.state
+  ```
+
+- [DataGridPro] Unification of the selection API parameters between `selectRow` and `selectRows`
+
+
+  ```diff
+  -selectRow: (id: GridRowId, isSelected?: boolean, allowMultiple?: boolean = false) => void;
+  +selectRow: (id: GridRowId, isSelected?: boolean, resetSelection?: boolean = false) => void;
+  ```
+
+- [DataGrid] Replace the `options` with direct prop reads
+
+  ```diff
+  -const { options } = useGridSlotComponentProps();
+  +const rootProps = useGridRootProps();
+  ``` 
+
+#### Changes
+
+- [DataGridPro] Only apply checkboxSelectionVisibleOnly when pagination is enabled (#2443) @flaviendelangle
+- [DataGridPro] Remove apiRef getState method (#2579) @flaviendelangle
+- [DataGrid] Add `disableVirtualization` prop (#2326) @m4theushw
+- [DataGrid] Add missing exports from param models (#2448) @flaviendelangle
+- [DataGrid] Add missing keys to the `classes` prop (#2458) @m4theushw
+- [DataGrid] Add row menu (#2385) @m4theushw
+- [DataGrid] Add zhCN localization (#2431) @wlf100220
+- [DataGrid] Add koKR localization (#2446) @zzossig
+- [DataGrid] Clean the selection public api (#2523) @flaviendelangle
+- [DataGrid] Do not call useGridColumnResize and useGridInfiniteLoader (#2580) @flaviendelangle
+- [DataGrid] Do not show right border of last column header when the its cells don't have it (#2444) @flaviendelangle
+- [DataGrid] Don't consider unselectable rows when selectionModel is used (#2464) @m4theushw
+- [DataGrid] Drop v4 support (#2515) @m4theushw
+- [DataGrid] Fire columnOrderChange event after state update (#2451) @flaviendelangle
+- [DataGrid] Fix TypeScript type error for toolbar components (#2393) @ZeeshanTamboli
+- [DataGrid] Fix navigation between column headers with rows filtered (#2440) @m4theushw
+- [DataGrid] Force scrollEndThreshold to undefined (#2574) @flaviendelangle
+- [DataGrid] Improve jaJP localization (#2502) @daikiojm
+- [DataGrid] Make `hideFooterRowCount` prop available only for DataGridPro (#2564) @ZeeshanTamboli
+- [DataGrid] Pressing `Escape` should close `GridColumnHeaderMenu` without errors (#2463) @DanailH
+- [DataGrid] Prevent scroll when selecting rows (#2558) @m4theushw
+- [DataGrid] Reduce specificity of GridToolbarContainer styles (#2462) @michaldudak
+- [DataGrid] Remove import to @material-ui/icons (#2576) @m4theushw
+- [DataGrid] Show page size controls on narrower screens (#2461) @michaldudak
+- [DataGrid] Vertically align column header icons (#2555) @oliviertassinari
+- [bug] The numeric filter operators are not handling '0' correctly (#2528) @flaviendelangle
```

Comment:
We haven't used `[bug]` before (I would even put it as `bugfix` as that si the label in GitHub). To have it as `bugfix` and be consistent some of the other entries needs to be updated as well.

Reply:
I can put `[DataGrid]` 
Making a release is nice, I'm noticing all the things I'm not doing correctly on my PR :smile:

Reply:
I think we can go with `DataGrid` and later maybe reevaluate.